spyder/widgets/ipythonconsole/shell.py

```diff
@@ -243,11 +243,11 @@ def handle_exec_method(self, msg):
                         self.sig_namespace_view.emit(view)
                     else:
                         view = None
-                elif 'get_var_properties' in method:
+                elif 'get_var_properties' in method and 'text/plain' in data:
```

Comment:
We need to add the `else` statement making the `view = None` for this clause too, right?